tests/api/retrieval.test.js

```diff
@@ -0,0 +1,339 @@
+import { createMocks } from "node-mocks-http";
+import client from "next-auth/client";
+import retrieval from "../../pages/api/retrieval";
+import { DynamoDBClient } from "@aws-sdk/client-dynamodb";
+import executeQuery from "@lib/integration/queryManager";
+import jwt from "jsonwebtoken";
+
+jest.mock("next-auth/client");
+jest.mock("@aws-sdk/client-dynamodb");
+jest.mock("@lib/integration/queryManager");
+
+jest.mock("@lib/integration/dbConnector", () => {
+  const mockClient = {
+    connect: jest.fn(),
+    query: jest.fn(),
+    end: jest.fn(),
+  };
+  return jest.fn(() => mockClient);
+});
+
+const dynamoClient = {
+  send: jest.fn(),
+};
+
+describe("/api/retrieval", () => {
+  beforeAll(() => {
+    process.env.TOKEN_SECRET = "gc-form-super-secret-code";
+  });
+  afterAll(() => {
+    delete process.env.TOKEN_SECRET;
+  });
+
+  it("Should return 400 status code b/c formID is an empty string", async () => {
+    const mockSession = {
```

Comment:
Do we need to mock a session for a get request ?

Reply:
No. Unnecessary or unused code was removed.